specification/elastic/resource-manager/Microsoft.Elastic/preview/2025-01-15-preview/examples/ConnectedPartnerResources_List.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "api-version": "2025-01-15-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "myResourceGroup",
+    "monitorName": "myMonitor"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "properties": {
+              "partnerDeploymentName": "deploymentname",
+              "partnerDeploymentUri": "https://examplessourl.com",
+              "azureResourceId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/monitors/myMonitor",
+              "location": "West US 2",
+              "type": "hostingType"
```

Comment:
>hostingType [](http://example.com/codeflow?start=23&length=11)

I see you are adding a new enum called hosting type in the new swagger, why are we not using it here? And why do we call it hostingtype in some places while type here?

Reply:
Is this different?